worlds/kh1/Options.py

```diff
@@ -0,0 +1,397 @@
+    TT   - Lionheart
+    WL   - Lady Luck
+    OC   - Olympia
+    DJ   - Jungle King
+    AG   - Three Wishes
+    MS   - Wishing Star
+    HT   - Pumpkinhead
+    NL   - Fairy Harp
+    HB   - Divine Rose
+    EotW - Oblivion
+    HAW  - Oathkeeper
+    
+    Note: Does not apply to Atlantica, the emblem and carousel chests in Hollow Bastion, or the Aero chest in Neverland currently.
+    """
+    display_name = "Keyblades Unlock Chests"
+
+class InteractInBattle(Toggle):
+    """
+    Allow Sora to talk to people, examine objects, and open chests in battle.
+    """
+    display_name = "Interact in Battle"
+
+class AdvancedLogic(Toggle):
+    """
+    If on, logic may expect you to do advanced skips like using Combo Master, Dumbo, and other unusual methods to reach locations.
+    """
+    display_name = "Advanced Logic"
+
+class ExtraSharedAbilities(Toggle):
+    """
+    If on, adds extra shared abilities to the pool.  These can stack, so multiple high jumps make you jump higher and multiple glides make you superglide faster.
+    """
+    display_name = "Extra Shared Abilities"
+
+class EXPZeroInPool(Toggle):
+    """
+    If on, adds EXP Zero ability to the item pool.  This is redundant if you are planning on playing on Proud.
+    """
+    display_name = "EXP Zero in Pool"
+
+class VanillaEmblemPieces(DefaultOnToggle):
+    """
+    If on, the Hollow Bastion emblem pieces are in their vanilla locations.
+    """
+    display_name = "Vanilla Emblem Pieces"
+
+class StartingWorlds(Range):
+    """
+    Number of random worlds to start with in addition to Traverse Town, which is always available.  Will only consider Atlantica if toggled, and will only consider End of the World if its unlock is set to "Item".
+    """
+    display_name = "Starting Worlds"
+    default = 0
+    range_start = 0
+    range_end = 10
+
+@dataclass
+class KH1Options(PerGameCommonOptions):
+    goal: Goal
+    end_of_the_world_unlock: EndoftheWorldUnlock
+    final_rest_door: FinalRestDoor
+    required_reports_eotw: RequiredReportsEotW
+    required_reports_door: RequiredReportsDoor
+    reports_in_pool: ReportsInPool
+    super_bosses: SuperBosses
+    atlantica: Atlantica
+    hundred_acre_wood: HundredAcreWood
+    cups: Cups
+    puppies: Puppies
+    starting_worlds: StartingWorlds
+    keyblades_unlock_chests: KeybladesUnlockChests
+    interact_in_battle: InteractInBattle
+    exp_multiplier: EXPMultiplier
+    advanced_logic: AdvancedLogic
+    extra_shared_abilities: ExtraSharedAbilities
+    exp_zero_in_pool: EXPZeroInPool
+    vanilla_emblem_pieces: VanillaEmblemPieces
+    donald_death_link: DonaldDeathLink
+    goofy_death_link: GoofyDeathLink
+    randomize_keyblade_stats: RandomizeKeybladeStats
+    bad_starting_weapons: BadStartingWeapons
+    keyblade_max_str: KeybladeMaxStrength
+    keyblade_min_str: KeybladeMinStrength
+    keyblade_max_mp: KeybladeMaxMP
+    keyblade_min_mp: KeybladeMinMP
```

Comment:
You'll probably want to switch the order of these so they display in the proper order on the site
```suggestion
    keyblade_min_str: KeybladeMinStrength
    keyblade_max_str: KeybladeMaxStrength
    keyblade_min_mp: KeybladeMinMP
    keyblade_max_mp: KeybladeMaxMP
```